platform/android/java_glue.cpp

```diff
@@ -1001,6 +1001,13 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_touch(JNIEnv *env, jo
 	*/
 }
 
+JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_hover(JNIEnv *env, jobject obj, jint p_device, jint p_x, jint p_y) {
```

Comment:
The `p_device` argument name is confusing since you're actually passing an event type.